autobahn/autobahn/wamp/types.py

```diff
@@ -434,7 +434,7 @@ def __init__(self,
         assert(timeout is None or (type(timeout) in list(six.integer_types) + [float] and timeout > 0))
         assert(disclose_me is None or type(disclose_me) == bool)
 
-        self.on_progress = on_progress
+        self.onProgress = on_progress
```

Comment:
There is other code that uses the camel-case, so wasn't sure what was correct. Also some examples. For example, here: https://github.com/tavendo/AutobahnPython/blob/master/autobahn/wamp/protocol.py#L660

So, everything should use on_progress instead?
I figured maybe the camelCase was the right way as it matched ISession-style API